### DISCUSSION TITLE
How to restrict a site in Cefsharp?

### DISCUSSION THREAD

**Discussion by Aravind-pro** [cefsharp/CefSharp#3680] [Q&A] [answered]
I would like to block some sites in my browser . Please help me

**Comment by amaitland** [comment, marked as answer]
You can use 
http://cefsharp.github.io/api/91.1.x/html/M_CefSharp_IRequestHandler_OnBeforeBrowse.htm 

The example at https://github.com/cefsharp/CefSharp/wiki/General-Usage#protocol-execution has the basic pattern you can follow.
